frontend/src/services/movies.ts

```diff
@@ -40,10 +41,68 @@ const filterMovies = async (pageNumber: number, genreId?: number | ""): Promise<
     }
 };
 
+const getMovieById = async (id: number): Promise<IMovie> => {
+    try {
+        const response = await axios.get(`${API_URL}/${id}`);
+        const movie = response.data;
+        return movie as IMovie;
+    } catch (error) {
+        console.error('Error fetching movies:', error);
+        throw error;
+    }
+};
+
+const addReview = (movieId:number, rating:number, comment: string, createdOn: string,accountId:number,movieTitle:string) => {
+    return axios
+      .post(`${API_URL}/${movieId}/reviews`, {
+        movieId,
+        rating,
+        comment,
+        createdOn,
+        accountId,
+        movieTitle
+      })
+      .then((response: { data: {}; }) => {
+        return response.data;
+      });
+  };
+
+  const getReviewsByMovieId = async (id: number): Promise<IReview[]>  => {
+    try {
+        const response = await axios.get(`${API_URL}/${id}/reviews`,{
+             
+                headers: {
+                    ...authHeader()
```

Comment:
In the next api version, we won't need the auth header because this endpoint should be available to be called by anyone

Reply:
then I will change it when the next api version will be av